src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -59,6 +59,28 @@ public static partial class PlatformDetection
         public static bool IsNetfx471OrNewer => false;
         public static bool IsNetfx472OrNewer => false;
 
+        public static bool IsDrawingSupported { get; } = GetGdiplusIsAvailable();
+
+        private static bool GetGdiplusIsAvailable()
+        {
+            IntPtr nativeLib;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                nativeLib = dlopen("libgdiplus.dylib", RTLD_NOW);
```

Comment:
You need to also move dlopen and RTLD_NOW from Drawing Helpers: https://github.com/dotnet/corefx/pull/33522/files#diff-3b9a146b103f18cfafd4c1d8d2971cedR76